bartop-ui/.storybook/preview-head.html

```diff
@@ -0,0 +1,7 @@
+<!-- this file lets us inject <head> tags for the storybook iframe -->
+<link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+<style>
+  #root {
+    font-family: 'Raleway', sans-serif;
```

Comment:
We do this so that `Storybook` fonts match the font we're using for the actual app